src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-enchant.py

```diff
@@ -59,4 +59,7 @@
         # In Macports are available mostly hunspell (myspell) and aspell dictionaries.
         libdir = os.path.dirname(libenchant)  # e.g. /opt/local/lib
         sharedir = os.path.join(os.path.dirname(libdir), 'share')  # e.g. /opt/local/share
-        datas.append((os.path.join(sharedir, 'enchant'), 'enchant/share/enchant'))
+        if os.path.exists(os.path.join(sharedir, 'enchant')):
+            datas.append((os.path.join(sharedir, 'enchant'), 'enchant/share/enchant'))
+        if os.path.exists(os.path.join(sharedir, 'enchant-2')):
+            datas.append((os.path.join(sharedir, 'enchant-2'), 'enchant/share/enchant'))
```

Comment:
The destination should probably `enchant/share/enchant-2' so that if both exist, they don't clobber each other?

Reply:
Good point. I just tested it out, and it appears my app is able to find the dictionaries without an issue.